org.azymohliad.Qwertone.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "org.azymohliad.Qwertone",
```

Comment:
this needs to be a real domain name that you control. in this case it could be com.gitlab.azymohliad.Qwertone for instance.

Reply:
Thank you! I was really confused about how to pick App ID properly. I wonder is real domain name required only by flathub, or generally by freedesktop standard?

Reply:
The standard requires it to be unique and reverse DNS style. A good way of making sure of that is if you actually own the domain name. Plus, at Flathub we like it as a hint as to the actual upstream if they weren't initially involved.